CODEOWNERS

```diff
@@ -26,6 +26,7 @@ lte/gateway/c/session_manager @themarwhal @uri200
 lte/gateway/c/oai @ssanadhya @ulaskozat @lionelgo @ardzoht
 lte/gateway/c/sctpd @ssanadhya @ulaskozat
 lte/gateway/c/connection_tracker @koolzz
+lte/gateway/c/li_agent @koolzz
```

Comment:
We should probably have another person to own the component. Not sure who makes sense though.

Reply:
I've seen a couple PRs that modify CODEOWNERS, @koolzz is there a reason you had to modify this for your development work? Did it hinder you in your fork in some way? We should figure this out so that people don't accidentally push CODEOWNERS changes (and e.g. I miss the CODEOWNERS change and LGTM allowing changes by accident).

Reply:
Not development specific reason, I just included it as it's a new service and I want to maintain it.